### ISSUE TITLE
Ask for a new open source icon library: EVA Icons

### ISSUE THREAD

**Issue by Pelirrojo** [react-icons/react-icons#253]
I would like to use this library: https://zakveo.github.io/eva-icons/#/?type=outline

Anyway, thanks for a great job!

It's really an improvement to include only a package with the most relevant and used libraries icons for your react app in a easy way.

**Comment by vpzomtrrfrt** [comment]
This link seems to be dead?

**Comment by Pelirrojo** [comment]
Yes, really it's an option in the web (two way of view the icons: outlined or filled)

* Site: https://akveo.github.io/eva-icons/
* NPM: https://www.npmjs.com/package/eva-icons
* Github Repo: https://github.com/akveo/eva-icons

The latest commit was in July of this year, so I think the project it's alive.

**Comment by pie6k** [comment]
I'd like to double it! It is an amazing set of icons.
